### DISCUSSION TITLE
Sponcer Profile isn't getting verified

### DISCUSSION THREAD

**Discussion by lkant8** [community/community#42124] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

My Github Sponcer's Profile isn't getting verified, says (Your Profile isn't live yet, We need More information to continue).

 Can anyone assist me please :)

**Comment by dthoma1** [comment, marked as answer]
If your profile is still not approved, you can [contact support](https://support.github.com/contact/account?type=github_sponsors.).

**Reply by lkant8** [reply]
verified thank you so much
